### ISSUE TITLE
ButtonGroup with 4 buttons. separator line missing 

### ISSUE THREAD

**Issue by xmlking** [themesberg/flowbite-svelte#241]
ButtonGroup with 4 buttons. separator line missing 
```
	<ButtonGroup>
		<Button outline on:click="{gotoPolicies}">
			<ArrowLeft size="18" class="mr-2 text-blue-500 dark:text-green-500" />
			Back
		</Button>
		{#if editable}
			<Button outline on:click="{deletePolicy}">
				<Archive size="18" class="mr-2 text-blue-500 dark:text-red-500" />
				Delete
			</Button>
			<Button outline on:click="{handleReset}">
				<Adjustments size="18" class="mr-2 text-blue-500 dark:text-green-500" />
				Reset
			</Button>
			<Button outline type="submit" disabled="{!$isValid}" class="{!$isValid ? 'cursor-not-allowed' : ''}">
				<CloudDownload size="18" class="mr-2 text-blue-500 dark:text-green-500" />
				{#if $isSubmitting}loading...{:else}Save{/if}
			</Button>
		{:else}
			<Button outline type="submit" disabled="{!$isValid}" class="{!$isValid ? 'cursor-not-allowed' : ''}">
				<Adjustments size="18" class="mr-2 text-blue-500 dark:text-green-500" />
				{#if $isSubmitting}loading...{:else}Create{/if}
			</Button>
		{/if}
	</ButtonGroup>
```
<img width="453" alt="image" src="https://user-images.githubusercontent.com/627951/186071506-b13ca695-0221-437f-8205-7e271693c07f.png">

**Comment by shinokada** [comment]
@jjagielka Can you have a look at this issue?

**Comment by jjagielka** [comment]
My bad. Fix provided #242
